core-models/src/main/scala/com/mesosphere/usi/core/models/PodRecord.scala

```diff
@@ -0,0 +1,9 @@
+package com.mesosphere.usi.core.models
+
+import java.time.Instant
+
+/**
+  * Snapshot of the Pod Spec state that needs to be persisted and can't be obtained from mesos.
```

Comment:
Please link pod spec:
```suggestion
  * Snapshot of the [[PodSpec]] state that needs to be persisted and can't be obtained from mesos.
```